### PR TITLE
Edits to Scheduled Pipelines doc

### PR DESCRIPTION
**Pull request by kbatuigas**
# Description
Some minor / additional clarifications.

# Reasons
Suggestions from review on previous PR: https://github.com/circleci/circleci-docs/pull/5939

### CODE REVIEW

**Comment by kbatuigas** [comment]
@rosieyohannan I've requested you as a reviewer so you can take a look at what's changed since the first publish!

**Review by devinmarieb — APPROVED** [review]
Thanks! @rosieyohannan has the final approval before merging :)

**Comment by rosieyohannan** [comment]
Thanks @kbatuigas I'll have a look now :-D

File: jekyll/_cci2/scheduled-pipelines.md
Comment (rosieyohannan) [inline jekyll/_cci2/scheduled-pipelines.md:31]:
```suggestion
- Take advantage of [auto-cancelling]({{site.baseurl}}/2.0/skip-build/#auto-cancelling).
```

File: jekyll/_cci2/scheduled-pipelines.md
Comment (rosieyohannan) [inline jekyll/_cci2/scheduled-pipelines.md:33]: Sounds like a good use case for an example for the config cookbook maybe. I'll make a ticket :-)

**Review by rosieyohannan — COMMENTED** [review]
Thanks so much for these clarifications! Just one interlink addition and then good to go 🎉

File: jekyll/_cci2/scheduled-pipelines.md
Comment (Plataformaalgoritmica) [reply jekyll/_cci2/scheduled-pipelines.md:33]: Great.

Comment (Plataformaalgoritmica) [reply jekyll/_cci2/scheduled-pipelines.md:33]: Hola Rosie, Gracias.

**Pull request merged**
